arm/Microsoft.Sql/managedInstances/readme.md

```diff
@@ -153,4 +153,4 @@ You can specify multiple user assigned identities to a resource by providing add
 - [Managedinstances/Keys](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Sql/2021-05-01-preview/managedInstances/keys)
 - [Managedinstances/Securityalertpolicies](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Sql/2017-03-01-preview/managedInstances/securityAlertPolicies)
 - [Managedinstances/Vulnerabilityassessments](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Sql/2021-02-01-preview/managedInstances/vulnerabilityAssessments)
-- [Roleassignments](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2021-04-01-preview/roleAssignments)
+- [Roleassignments](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/roleAssignments)
```

Comment:
This PR also contains API version removal from the template reference link

Reply:
I know. I re-ran the generation of the readme's instead of performing changes manually. Apparently, the URL isn't actually available. A quick check confirmed this suspicion.